src/logic/map_objects/world/world.cc

```diff
@@ -113,7 +102,11 @@ const DescriptionMaintainer<ImmovableDescr>& World::immovables() const {
 
 void World::add_immovable_type(const LuaTable& table) {
 	i18n::Textdomain td("world");
-	immovables_->add(new ImmovableDescr(_(table.get_string("descname")), table, *this));
+	std::vector<std::string> attributes;
+	if (table.has_key("attributes")) {
+		attributes = table.get_table("attributes")->array_entries<std::string>();
+	}
+	immovables_->add(new ImmovableDescr(_(table.get_string("descname")), table, attributes, *this));
```

Comment:
Isn't the descname already translated by the lua script? No need to translate it again…

Reply:
We'll have to add push_textdomain/pop_textdomain to all init.lua files to test if that would work.

-1 for doing that in this branch. We should go over these though once all the refactoring of the tribes and world has been done.